SU-HSDO/suhumsci
# SHS-5772 - Regression: External Links on Postcards are wrong color

# READY FOR REVIEW

## Summary
Fix regression in external links on Postcards title

## Need Review By (Date)
08/09

## Urgency
medium

## Steps to Test
1. Go to the following sites
  
  - https://economics.suhumsci.loc/undergraduate/honors-program (postcards, Colorful Lake)
  - https://chemistry.suhumsci.loc/about/culture (view Traditional, Cardinal)
  - https://archaeology.suhumsci.loc/ (publications view, Colorful Ocean)
  - https://statistics.suhumsci.loc/industrial-affiliates/industrial-affiliates-program (postcards, Colorful Mountain)
  - https://sts.suhumsci.loc/news-publications/sts-professional-journals (view Colorful Cardinal)

2. Check that the external links are now lookins as expected (colour is black text with bottom border)
3. On Colorful and Traditional
4. Go to `/components/collections-cards/collections-postcards`
5. Edit some of them and add internal and external links for all the variations of Postcards (except the vertical linked card)
6. Check the look the same as expected
7. Also test everything in multiple colour palettes (pay attention to the Warbler color pairing in Traditional themes)

## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Humsci Basic PR Checklist](https://github.com/SU-HSDO/suhumsci/blob/develop/docs/HumsciBasicPRChecklist.md)


---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1207996019673293
